src/OpenTelemetry.Exporter.Geneva/Internal/ConnectionStringBuilder.cs

```diff
@@ -75,6 +75,12 @@ public string PrivatePreviewEnableTraceLoggingDynamic
         set => this._parts[nameof(this.PrivatePreviewEnableTraceLoggingDynamic)] = value;
     }
 
+    public string PrivatePreviewOtlpProtobufMetricExporter
+    {
+        get => this._parts.TryGetValue(nameof(this.PrivatePreviewOtlpProtobufMetricExporter), out var value) ? value : null;
```

Comment:
We probably would have to change the name to something else such as `EnableOtlpProtobufEncoding`.